content/blogposts/2024/cody-vscode-1-10-0-release.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Cody for VS Code v1.10.0 release"
+authors:
+  - name: Kalan Chan
+    url: https://handbook.sourcegraph.com/team/#kalan-chan
+publishDate: 2024-03-20T10:00-01:00
+description: "Cody for VS Code v1.10.0 is now available, this release includes support for Claude 3 Haiku, several improvements to doc string generation, and debugging."
+tags: [blog]
+slug: "cody-vscode-1-10-0-release"
+published: true
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/cody-vscode-1.10.0/cody-vscode-1.10.0-og-image.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/cody-vscode-1.10.0/cody-vscode-1.10.0-og-image.png
+--- 
+
+[Cody for VS Code](https://marketplace.visualstudio.com/items?itemName=sourcegraph.cody-ai) v1.10.0 is now available, this release includes support for Claude 3 Haiku, several improvements to doc string generation, and debugging. 
+
+
+## Support links for Cody Pro and Enterprise
+
+We have made it easier for Cody Pro and Enterprise users to get assistance when needed, support links are now more prominently displayed so users can easily access help. Cody Pro and Enterprise users can now get more timely responses by filling out a support form. 
+
+<Figure
+  src="https://storage.googleapis.com/sourcegraph-assets/blog/cody-vscode-1.10.0/support1.png"
+  alt="Support link in the settings panel"
+/>
+
+<Figure
+  src="https://storage.googleapis.com/sourcegraph-assets/blog/cody-vscode-1.10.0/support2.png"
+  alt="Support link in the Cody settings panel"
+/>
+
+## Improved Document Command
+
+Generating documentation from code has been improved to automatically detect documentable code blocks and provide hints to quickly generate doc strings. When your cursor is on a documentable symbol in JavaScript, TypeScript, Go or Python, a "Opt/Alt+D to Document" ghost text will appear. Pressing Opt/Alt+D will immediately generate documentation for that code block.
+
+<Figure
+  src="https://storage.googleapis.com/sourcegraph-assets/blog/cody-vscode-1.10.0/document-command.png"
+  alt="Document Command"
+/>
+
+## Claude 3 Haiku is now available for Cody Pro 
```

Comment:
If we mention Haiku should we also mention that we've updated GPT 4 Turbo to the latest version?

Reply:
good point!